wire-ios-testing/Source/Public/DateProviders/MockStaticDateProvider.swift

```diff
@@ -19,10 +19,13 @@
 import Foundation
 import WireSystem
 
-extension DateProviding where Self == MockStaticDateProvider {
-    public static func mock(_ now: Date) -> Self { .init(now: now) }
+extension CurrentDateProviding where Self == MockStaticDateProvider {
+
+    public static func mock(_ now: Date) -> Self {
+        .init(now: now)
+    }
```

Comment:
I'm not sure if this is used at all, can't find it in the project. And why can't the user just call `MockStaticDateProvider.now`?

Reply:
Deleted.